test/specs/test.e2e.js

```diff
@@ -0,0 +1,15 @@
+import { expect } from '@wdio/globals';
+import LoginPage from '../pageobjects/login.page.js';
+import SecurePage from '../pageobjects/secure.page.js';
+
+describe('My Login application', () => {
+  it('should login with valid credentials', async () => {
+    await LoginPage.open();
+
+    await LoginPage.login('tomsmith', 'SuperSecretPassword!');
```

Comment:
Let's move user credentials to the separate file, i.e. `test_data/users.js`

Reply:
Test was created with wdio intalation. However I have modify it for our test env.